api/scripts/export-ngrok.py

```diff
@@ -10,7 +10,7 @@
         if tunnel['proto'] != 'https':
             continue
 
-        if tunnel['config']['addr'] != 'localhost:6011':
+        if tunnel['config']['addr'] != 'http://localhost:6011':
```

Comment:
has ngrok changed? this is hostname without schema for me.

also this will never resolve because we've skipped unless https in the check above

Reply:
i think it changed because it didn't work for me unless i made this change.

with this change in place:

```
$ ngrok --version
ngrok version 2.3.35
$ pipenv run python scripts/export-ngrok.py
Loading .env environment variables…
https://redacted.ngrok.io
```

if i revert this change:

```
$ pipenv run python scripts/export-ngrok.py
Loading .env environment variables…
```

i.e. no output